docs/docs/security/cves.mdx

```diff
@@ -4,15 +4,30 @@ hide_title: true
 sidebar_position: 2
 ---
 
+#### Version 3.0.2, 2.1.3
+
+| CVE            | Title                                                       |                   Affected |
+|:---------------|:------------------------------------------------------------|---------------------------:|
+| CVE-2023-46104 | Allows for uncontrolled resource consumption via a ZIP bomb | < 2.1.3, >= 3.0.0, < 3.0.2 |
+| CVE-2023-49736 | SQL Injection on where_in JINJA macro                       | < 2.1.3, >= 3.0.0, < 3.0.2 |
+| CVE-2023-49734 | Privilege Escalation Vulnerability                          | < 2.1.3, >= 3.0.0, < 3.0.2 |
+
+
 #### Version 3.0.0
 
 | CVE            | Title                                                                   | Affected |
 |:---------------|:------------------------------------------------------------------------|---------:|
 | CVE-2023-42502 | Open Redirect Vulnerability                                             |  < 3.0.0 |
-| CVE-2023-42504 | Lack of rate limiting allows for possible denial of service             |  < 3.0.0 |
 | CVE-2023-42505 | Sensitive information disclosure on db connection details               |  < 3.0.0 |
 
 
+#### Version 2.1.3
+
+| CVE            | Title                                                                   | Affected |
+|:---------------|:------------------------------------------------------------------------|---------:|
+| CVE-2023-42504 | Lack of rate limiting allows for possible denial of service             |  < 2.1.3 |
+
```

Comment:
you can also add if needed
[CVE-2023-30608](https://github.com/advisories/GHSA-rrm6-wvj7-cwh2)
[CVE-2023-30861](https://github.com/advisories/GHSA-m2qf-hxjv-5gpq)

Reply:
I am not, we should track and patch these in an automated fashion